fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DimensionsServlet.java

```diff
@@ -332,6 +326,27 @@ public Response getDimensionRows(
         }
     }
 
+    /***
+     * Get the pagination of dimension rows.
+     *
+     * @param apiRequest The apiRequest
+     * @param searchProvider The searchProvider
+     * @param paginationParameters The pagination parameters
+     * @return pagination of dimensionRow
+     */
+    protected Pagination<DimensionRow> getPagedRows(
```

Comment:
@michael-mclawhorn Do you think it makes more sense to move this method to the skeletal implementation which is `EndpointServlet`? Other servlet might find it useful in the future.